web/static/js/components/surveys/SurveyCard.jsx

```diff
@@ -77,106 +75,337 @@ class SurveyCard extends Component<any> {
 
   deletable() {
     const { survey, readOnly } = this.props
-    if (readOnly) return false
-    return survey.isDeletable
+    return !readOnly && survey.isDeletable
   }
 
   movable() {
     const { survey, readOnly } = this.props
-    if (readOnly) return false
-    return survey.isMovable
+    return !readOnly && survey.isMovable
+  }
+
+  actionable() {
+    return this.deletable() || this.movable()
+  }
+
+  render() {
+    const { survey, t, dispatch } = this.props
+
+    let actions = []
+    if (this.movable()) actions.push({ name: t('Move to'), func: this.askMoveSurvey })
```

Comment:
I couldn't move a survey anymore, so I changed the callback back to `moveSurvey()` (which exists) and it worked!

```suggestion
    if (this.movable()) actions.push({ name: t('Move to'), func: this.moveSurvey })
```

Reply:
yeah! A replace error 🙈 
I'm rewriting this to a `ask<action>` and `confirm<action>` pattern (like in `PanelSurveyCard`)